test/testcafe/framework/page-objects/EnrollEmailPageObject.js

```diff
@@ -74,11 +74,6 @@ export default class EnrollAuthenticatorPhonePageObject extends BasePageObject {
   async verificationLinkTextExists(emailAddress) {
     const text = await this.form.getSubtitle();
 
-    if (userVariables.v3) {
-      return text.includes('We sent you a verification email.');
-    }
-    else {
-      return text.includes(`We sent an email to ${emailAddress}.`);
-    }
```

Comment:
For some reason I have a memory that we intentionally changed the text for this but I can't recall. Do you have any recollection of if we made a product change here?

Reply:
I think in the early days of the widget, we had a different message (when we were trying to introduce new keys before settling on going for parity first). But since then we made it match v2 (or tried), this seems like it slipped through the cracks, when I compared the v2 view to our transformer, we were trying to grab the email from the profile (within the authenticator) but it doesn't exist there for enroll flow. 

v2 pulls it from the context and the test asserts it this way too :(